src/status_im/ui/screens/wallet/send/views.cljs

```diff
@@ -130,7 +130,7 @@
          [render-contact to from-chat?]]]
        [toolbar/toolbar
         {:center
-         {:label               :t/wallet-send
+         {:label               :t/next
```

Comment:
we can't just change label, because it can be "send" when you're sending request

Reply:
when we send tx in chat

Reply:
thanks!

Reply:
Updated logic